sdk/communication/communication-phone-numbers/test/public/ctor.oc.spec.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I'd name this file as `ctor.operatorConnect.spec.ts` for clarity. Unless `OC` is a widely known acronym.

Reply:
done